### PR TITLE
Remove Provisioning from bases

### PR DESCRIPTION
**Pull request by natifridman**
Since `Provisioning` is a global configuration, there is no need to add it to every cluster manifest.
Removing Provisioning from cluster bases.

Depends on redhat-partner-solutions/vse-carslab-hub/pull/16

### CODE REVIEW

**Comment by jnunyez** [comment]
> Since `Provisioning` is a global configuration, there is no need to add it to every cluster manifest. Removing Provisioning from cluster bases.
> 
> Depends on [redhat-partner-solutions/vse-carslab-hub/pull/16](https://github.com/redhat-partner-solutions/vse-carslab-hub/pull/16)

Just for everybody to follow and sync since this has an impact in all kind of cluster deployments (also SNO). @novacain1 @hhamid93  We are gonna remove all ocurrences of Provisioning in vse-ocp-bases. We assume that there is one Provisioning object with disableVirtualmediaTLS set to **true** if needed when configuring the Hub cluster. 
All the managed clusters will leverage that object with that configuration no matter the server model.

**Comment by jnunyez** [comment]
> > Since `Provisioning` is a global configuration, there is no need to add it to every cluster manifest. Removing Provisioning from cluster bases.
> > Depends on [redhat-partner-solutions/vse-carslab-hub/pull/16](https://github.com/redhat-partner-solutions/vse-carslab-hub/pull/16)
> 
> Just for everybody to follow and sync since this has an impact in all kind of cluster deployments (also SNO). @novacain1 @hhamid93 We are gonna remove all ocurrences of Provisioning in vse-ocp-bases. We assume that there is one Provisioning object with disableVirtualmediaTLS set to **true** if needed when configuring the Hub cluster. All the managed clusters will leverage that object with that configuration no matter the server model.

Other than that LGTM let's merge it after we merge  https://github.com/redhat-partner-solutions/vse-carslab-hub/pull/16

**Review by jnunyez — APPROVED** [review]

**Review by novacain1 — APPROVED** [review]
LGTM, until we have any movement on this from an RFE perspective (in progress).

**Pull request merged**
